sui-execution/latest/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -1347,7 +1351,23 @@ mod checked {
         let accumulator_events = accumulator_events
             .into_iter()
             .map(|accum_event| match accum_event.value {
-                MoveAccumulatorValue::MoveValue(_, _, _) => todo!(),
+                MoveAccumulatorValue::U64(amount) => {
+                    let value = AccumulatorValue::Integer(amount);
+                    let address = AccumulatorAddress::new(
+                        accum_event.target_addr.into(),
+                        vm.get_runtime()
+                            .get_type_tag(&accum_event.target_ty)
+                            .unwrap(),
+                    );
```

Comment:
See other comments, but this will not work long term. You need to generate any of this information in the object runtime or in the native.